### PR TITLE
Upgrade of reviewers to approvers, remove inactives from reviewer list

### PR DESCRIPTION
**Pull request by a-roberts**
Upgrades both @eddycharly and @CarolynMabbott to be approver and removes now inactive reviewers that were often automatically popping up (thanks for all the good work!)

Hopefully this syntax is ok, let's see

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/dashboard/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Comment by a-roberts** [comment]
@eddycharly @AlanGreene FYI

**Comment by eddycharly** [comment]
Thank you !

/meow party

**Comment by a-roberts** [comment]
@eddycharly thank *you*, it's better for the project this way - I'm working on other things atm so while happy to provide oversight and support where I can, this is the right thing to do IMO so the project continues to grow 🥳

File: OWNERS [hunk 1]
```diff
@@ -5,14 +5,12 @@
...
 - a-roberts
 - alangreene
+- eddycharly
 - mnuttall
 - skaegi
 - steveodonovan
 - dibbles
+- carolynmabbott
 
 reviewers:
```
Comment (AlanGreene) [inline OWNERS:15]: I think we need to remove this entirely as in the website and cli repos

File: OWNERS [hunk 2]
```diff
@@ -5,14 +5,12 @@
...
-- akihikokuroda
-- carolynmabbott
-- megan-wright
-- ncskier
-- eddycharly
+
```
Comment (AlanGreene) [inline OWNERS:16]: Can we add an 'Alumni' section like in some of the other projects? https://github.com/tektoncd/pipeline/blob/0b345b56bda47c4542519102527889f7d0777a33/OWNERS_ALIASES#L30

File: OWNERS (hunk 1 above)
Comment (a-roberts) [reply OWNERS]: Done

File: OWNERS (hunk 2 above)
Comment (a-roberts) [reply OWNERS]: Great idea, added

**Pull request merged**